app/models/case_assignment.rb

```diff
@@ -36,8 +36,8 @@ def casa_case_and_volunteer_must_belong_to_same_casa_org
 #  hide_old_contacts :boolean          default(FALSE)
 #  created_at        :datetime         not null
 #  updated_at        :datetime         not null
-#  casa_case_id      :integer          not null
-#  volunteer_id      :integer          not null
+#  casa_case_id      :bigint           not null
```

Comment:
this is an interesting diff, I'd rather not have this but I know it goes back and forth constantly